modules/fb_instant_articles_views/src/Plugin/views/filter/ValidFacebookInstantArticles.php

```diff
@@ -64,68 +66,70 @@ public function init(ViewExecutable $view, DisplayPluginBase $display, array &$o
 
   /**
    * {@inheritdoc}
-   *
-   *
    */
   public function query() {
-
     $this->enabledNodeBundlesSetValues();
     parent::query();
   }
 
   /**
-   * Set the values for this filter to all node bundles that implement custom settings for the view mode
+   * Set the values for this filter.
    *
-   * This uses an unreliable method for detecting custom settings for node bundles.  The method is to check for a config
-   * Entity for the entity display settings, for the bundle/view mode.  Essentially this looks for a yml file/record
-   * for the custom settings, and one is found, the bundle is considered FIA active.
+   * This applies to all node bundles that implement custom settings for the
+   * view mode.
    *
+   * This uses an unreliable method for detecting custom settings for node
+   * bundles.  The method is to check for a config entity for the entity display
+   * settings, for the bundle/view mode.  Essentially this looks for a yml
+   * file/record for the custom settings, and one is found, the bundle is
+   * considered FIA active.
    */
   protected function enabledNodeBundlesSetValues() {
-    /**
-     * @var \Drupal\Core\Entity\EntityTypeBundleInfo $entityBundleInfo
+    /*
+     * @var \Drupal\Core\Entity\EntityTypeBundleInfo $entity_bundle_info
      *  entity_type.bundle.info
      */
-    $entityBundleInfo = \Drupal::service('entity_type.bundle.info');
+    $entity_bundle_info = \Drupal::service('entity_type.bundle.info');
```

Comment:
Drat.  The create()/__construct() thing isn't working here, so tried a factory like you did for DrupalClient.  What a surprise, can't get that working, either.